README.md

```diff
@@ -4,3 +4,15 @@ Common build helpers for sclorg containers
 This repository is aimed to be added as git submodule into particular
 containers' source repositories.  By default, the path to submodule should be
 named 'common'.
+
+Regression tests
+----------------
+
+Just run `make check`
+
+Dependencies for testsuite:
+
+- docker
+- docker-squash (version 1.0.5)
+- git
+- make
```

Comment:
Also `source-to-image` and `go-md2man` is needed for testsuites of tested images)